docs/test-configuration-options.md

```diff
@@ -0,0 +1,884 @@
+---
+id: test-configuration-options
+title: Test Configuration Options
+sidebar_label: Test Configuration Options
+---
+
+export const Highlight = ({children, color}) => ( <span style={{
+      backgroundColor: color,
+      borderRadius: '2px',
+      color: '#fff',
+      padding: '0.2rem',
+    }}>{children}</span> );
+
+In this topic, you'll find a list of Sauce Labs test configuration options. You can use the [Sauce Labs Platform Configurator](https://wiki.saucelabs.com/display/DOCS/Platform+Configurator) to generate the correct configuration of testing options for your choice of Appium or Selenium tests in your preferred programming language.
+
+We always recommend using the latest released version of Selenium, but to ensure W3C WebDriver compliance, make sure to use Selenium version 3.11 or later. Sauce Labs determines W3C sessions with the presence of `sauce:options` capabilities and [generic W3C WebDriver-compliant capabilities](https://www.w3.org/TR/webdriver1/#capabilities). See [W3C Capabilities Support](https://wiki.saucelabs.com/pages/viewpage.action?pageId=78414463) for more information.
+
+
+## Desktop Browser W3C Capabilities (Required)
+
+Below are the W3C WebDriver primary test configuration settings for Sauce Labs desktop browser tests.
+
+### `browserName`
+__Description__: identifies the user agent.
+
+__Value Type__: string.
+
+__Example__: `"browserName": "firefox"`
+<br/>
+
+### `browserVersion`
+__Description__: identifies the version of the browser you want to use in your test.
+
+__Value Type__: string.
+
+__Example__: `"browserVersion": "latest"`
+
+:::tip Default to Latest Version of Chrome, Firefox, or Microsoft Edge
+
+If you want to use the latest stable version of Google Chrome or Firefox that Sauce supports, you can use `"browserVersion": "latest"`. You can also use `"browserVersion": "latest-1"` or `"browserVersion": "latest-2"`, etc. to request the next most recent versions of a browser. For example, if the latest stable version of Chrome is 73, you can request `"latest-2"` to use Chrome 71.
+:::
+
+:::tip Microsoft Edge versions
+
+Microsoft Edge has two version numbers, the browser application version and the EdgeHTML rendering engine version. For example, the current stable release of Edge as of November 2019 has the browser application version 44.17763 and the EdgeHTML version 18.17763. For more details, see the [Microsoft Edge Wikipedia page](https://en.wikipedia.org/wiki/Microsoft_Edge).
```

Comment:
@titusfortner Comments from Diego and Michelle:

- In my opinion, we should replace this content with the new Edge versions, and also add a comment for IE and Safari.
- I don't think we have the need to keep this Edge information regarding legacy versions. Should just add edge to the above section about using "latest". I would like to possibly see something here or in the platformName section about using latest for Safari versions and the ability to not pin to a specific MacOS version in order to get the latest safari version even if it's on a new macOS version.

Reply:
This is my proposed wording: 
https://github.com/saucelabs/sauce-docs/pull/368/files#diff-0fc2069221eb88630ed37f491c9da96f8a956c53a37c3a6f2cbb4c7e8d16cf70R44-R53